migrations/sqls/20190806211241-client-interaction-up.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE IF NOT EXISTS clientInteractions (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    clientInteractionType ENUM (
+        'clientInteractionCreated',
+        'clientInteractionUpdated',
+        'clientInteractionDeleted'
+    ) NOT NULL,
+    clientId INT NOT NULL,
+    title VARCHAR(255) NOT NULL,
+    serviceId INT NOT NULL,
+    interactionType ENUM (
+        'in pesron',
+        'by phone',
+        'workshop/talk',
+        'one on one/light touch',
+        'consultation'
+    ) NOT NULL,
+    description MEDIUMTEXT,
+    date DATE DEFAULT CURRENT_DATE NOT NULL,
+    duration TIME DEFAULT '0:00:00' NOT NULL,
+    location ENUM (
+        'CU Office',
+        'Consulate Office',
+        'Community Event'
+    ) NOT NULL,
+    canModify BOOLEAN DEFAULT false NOT NULL,
```

Comment:
This doesn't need to be stored in the database, nor does it need to be in the client interactions API.

The canModify field in the client logs API is a boolean that is false for things like "basic information updated" but true for "case note" and "client interaction." We only need it in the client logs api, not in the client interaction api.

See https://github.com/JustUtahCoders/comunidades-unidas-internal/blob/9c8ec0fff60ef58fa220283e5abb9c4b925c14c4/backend/apis/clients/client-logs/activity-log.utils.js#L47 for where this is being calculated for the client logs api